### PR TITLE
Add elevator pitch

### PR DESCRIPTION
**Pull request by juliusgeo**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -11,6 +11,21 @@ About
 =====
 This package provides an ``ExplainCollection`` class that allows PyMongo's Collection methods to be explained_
 
+PyMongoExplain greatly simplifies the amount of effort needed to explain commands.
+For example, suppose we wanted to explain the following ``update_one``: ::
```
Comment (ShaneHarvey) [inline README.rst:15]: Here and below, I think you can replace `: ::` with just `::`

Comment (juliusgeo) [reply README.rst:15]: Done.

File: README.rst [hunk 2]
```diff
@@ -11,6 +11,21 @@ About
 =====
 This package provides an ``ExplainCollection`` class that allows PyMongo's Collection methods to be explained_
 
+PyMongoExplain greatly simplifies the amount of effort needed to explain commands.
+For example, suppose we wanted to explain the following ``update_one`` ::
```
Comment (ShaneHarvey) [inline README.rst:15]: RST is very fickle. You need to remove the space before `::` or else the `:` won't be rendered.

File: README.rst [hunk 3]
```diff
@@ -11,6 +11,21 @@ About
 =====
 This package provides an ``ExplainCollection`` class that allows PyMongo's Collection methods to be explained_
 
+PyMongoExplain greatly simplifies the amount of effort needed to explain commands.
+For example, suppose we wanted to explain the following ``update_one``::
+
+    collection.update_one({"quantity": 1057, "category": "apparel"},{"$set": {"reorder": True}})
+
+
+Before PyMongoExplain, one would need to convert the update_one into the equivalent MongoDB command ::
```
Comment (ShaneHarvey) [inline README.rst:20]: Please make the whitespace change throughout the whole file.

**Pull request merged**